### PR TITLE
Scale BaseAxis properly

### PR DESCRIPTION
**Pull request by ahcorde**
While I was adding the origin axes to ignition gazebo I checked that the scale for a BaseAxis is wrong. Current implementation:

scale_wrong

Overriding the scale methods:

scale_ok

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: include/ignition/rendering/base/BaseAxisVisual.hh [hunk 1]
```diff
@@ -37,6 +37,18 @@ namespace ignition
       public: virtual ~BaseAxisVisual();
 
       public: virtual void Init();
+
+      public: virtual void SetLocalScale(double _x, double _y, double _z) override;
+
+      public: virtual void SetLocalScale(double _scale) override;
+
+      public: virtual void SetLocalScale(const math::Vector3d &_scale) override;
```
Comment (iche033) [inline include/ignition/rendering/base/BaseAxisVisual.hh:45]: I think you should be able to just override this version with `math::Vector3d` arg. The other two just calls this function so can be removed.

File: include/ignition/rendering/base/BaseAxisVisual.hh [hunk 2]
```diff
@@ -51,25 +63,52 @@ namespace ignition
...
+      xArrow->SetLocalScale(_scale, _scale, _scale);
+      yArrow->SetLocalScale(_scale, _scale, _scale);
+      zArrow->SetLocalScale(_scale, _scale, _scale);
+    }
+
+    //////////////////////////////////////////////////
+    template <class T>
+    void BaseAxisVisual<T>::SetLocalScale(const math::Vector3d &_scale)
+    {
+      xArrow->SetLocalScale(_scale.X(), _scale.Y(), _scale.Z());
```
Comment (iche033) [inline include/ignition/rendering/base/BaseAxisVisual.hh:88]: how about:

```
      for (unsigned int i = 0; i < this->ChildCount(); ++i)
        this->ChildByIndex(i)->SetLocalScale(_scale);
```

this way we don't need to declare the private `xArrow`, `yArrow`, `zArrow` variables

**Comment by ahcorde** [comment]
This PR builds on top of this other PR https://github.com/ignitionrobotics/ign-rendering/pull/87

**Comment by iche033** [comment]
looks good, updated change log in 71f4e44

**Comment by ahcorde** [comment]
there is an issue in one of the test https://build.osrfoundation.org/job/ignition_rendering-ci-pr_any-ubuntu_auto-amd64/623/testReport/junit/(root)/AxisVisual_AxisVisualTest/AxisVisual_ogre/

 ```plain
/var/lib/jenkins/workspace/ignition_rendering-ci-pr_any-ubuntu_auto-amd64/ign-rendering/src/AxisVisual_TEST.cc:65
Expected equality of these values:
  math::Vector3d(0.2, 0.3, 0.4)
    Which is: 0.2 0.3 0.4
  visual->LocalScale()
    Which is: 1 1 1
```

**Comment by ahcorde** [comment]
CI is green except on windows

**Comment by iche033** [comment]
tests on windows are not fully working yet so the failures are expected

**Comment by chapulina** [comment]
> tests on windows are not fully working yet so the failures are expected

Just an FYI that the [buildfarmer's script](https://github.com/osrf/buildfarmer/issues/49#issuecomment-636169565) indicates that these 2 failures are new on Windows:

```
    AxisVisual/AxisVisualTest.AxisVisual/ogre
    ArrowVisual/ArrowVisualTest.ArrowVisual/ogre
```

**Pull request merged**